blockchain/state_manager/src/chain_rand.rs

```diff
@@ -232,7 +232,9 @@ where
         round: ChainEpoch,
         entropy: &[u8],
     ) -> anyhow::Result<[u8; 32]> {
-        task::block_on(self.get_chain_randomness_v2(&self.blks, pers, round, entropy))
+        Runtime::new()
```

Comment:
just to be sure, is it equivalent in terms of operations? That is to say, did previous implementation instantiated a new runtime as well?

Reply:
They're equivalent. `task::block_on` creates a new Runtime and calls `rt.block_on`.